cmd/version.go

```diff
@@ -0,0 +1,23 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
```

Comment:
Do I miss something or do we need `//go:generate ../mkversions.sh` here? And `go generate ./cmd` in debian/rules?